### ISSUE TITLE
Compatibility certification request for Open Liberty for Jakarta EE 11, Core Profile

### ISSUE THREAD

**Issue by brideck** [jakartaee/platform#975]
- [x] Organization Name ("Organization") and, if applicable, URL:<br/>
  [IBM Corporation](https://www.ibm.com)
- [x] Product Name, Version and download URL (if applicable):<br/>
  [Open Liberty, 24.0.0.11-beta](https://public.dhe.ibm.com/ibmdl/export/pub/software/openliberty/runtime/beta/24.0.0.11-beta/openliberty-24.0.0.11-beta.zip)
- [x] Specification Name, Version and download URL:<br/>
  [Jakarta EE Core Profile, 11](https://jakarta.ee/specifications/coreprofile/11)
- [x] TCK Version, digital SHA-256 fingerprint and download URL:<br/>
  [Jakarta EE Core Profile TCK 11.0.0 (Staged Release)](https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee11/staged/eftl/jakarta-core-profile-tck-11.0.0.zip)
 `cd5d039adcb17a626845ee43c03e4ff11fbdec6738afb251bc3b1778c389e1c2`
- [x] Public URL of TCK Results Summary:<br/>
  [Java 17 Test Results](https://openliberty.io/certifications/jakartaee/11/coreprofile/24.0.0.11-beta-Java17-TCKResults.html)
  [Java 21 Test Results](https://openliberty.io/certifications/jakartaee/11/coreprofile/24.0.0.11-beta-Java21-TCKResults.html)
- [x] Any Additional Specification Certification Requirements:<br/>
  Jakarta Annotations
  [jakarta-annotations-tck-3.0.0.zip](https://download.eclipse.org/jakartaee/annotations/3.0/jakarta-annotations-tck-3.0.0.zip)
 `9421c6ca66274d32dfb408848f75a42d57f120599fe0d8403c5c5c1141d5ac4d`

  Jakarta Contexts and Dependency Injection
  [cdi-tck-4.1.0-dist.zip](https://download.eclipse.org/jakartaee/cdi/4.1/cdi-tck-4.1.0-dist.zip)
 `446029ee1ce694d2a9ae8893d16be7afd7e1c0ed8705064b7095af174cf97ea0`

  Jakarta Dependency Injection
  [jakarta.inject-tck-2.0.2-bin.zip](https://download.eclipse.org/jakartaee/dependency-injection/2.0/jakarta.inject-tck-2.0.2-bin.zip)
 `23bce4317ca061c3de648566cdf65c74b57e1264d6891f366567955d6b834972`

  Jakarta JSON Binding
  [jakarta-jsonb-tck-3.0.0.zip](https://download.eclipse.org/jakartaee/jsonb/3.0/jakarta-jsonb-tck-3.0.0.zip)
 `954fd9a3a67059ddeabe5f51462a6a3b542c94fc798094dd8c312a6a28ef2d0b`

  Jakarta JSON Processing
  [jakarta-jsonp-tck-2.1.1.zip](https://download.eclipse.org/jakartaee/jsonp/2.1/jakarta-jsonp-tck-2.1.1.zip)
 `949f203de84deffa8c7892b555918e42f1dd220ccb7b6800741ea58af62737c1`

  Jakarta RESTful Web Services
  [jakarta-restful-ws-tck-4.0.1.zip](https://download.eclipse.org/jakartaee/restful-ws/4.0/jakarta-restful-ws-tck-4.0.1.zip)
 `b6290c1b5b3d2fdd9cc700a999243492a7e27b94a9b6af1974ff4dc5bfbf98f2`
- [x] Java runtime used to run the implementation:<br/>
  Java 17:
  java version "17.0.12" 2024-07-16
  IBM Semeru Runtime Certified Edition 17.0.12.1 (build 17.0.12+7)
  Eclipse OpenJ9 VM 17.0.12.1 (build openj9-0.46.1, JRE 17 Linux amd64-64-Bit Compressed References 20240716_775 (JIT enabled, AOT enabled)
  OpenJ9   - 4760d5d320
  OMR      - 840a9adba
  JCL      - 9882134b8c0 based on jdk-17.0.12+7)

  Java 21:
  java version "21.0.4" 2024-07-16 LTS
  IBM Semeru Runtime Certified Edition 21.0.4.1 (build 21.0.4+7-LTS)
  Eclipse OpenJ9 VM 21.0.4.1 (build openj9-0.46.1, JRE 21 Linux amd64-64-Bit Compressed References 20240716_259 (JIT enabled, AOT enabled)
  OpenJ9   - 4760d5d320
  OMR      - 840a9adba
  JCL      - 6232a5d28a1 based on jdk-21.0.4+7)
- [x] Summary of the information for the certification environment, operating system, cloud, ...:<br/>
  Linux (5.15.0-118-generic; amd64)
- [x] By checking this box I acknowledge that the Organization I represent accepts the terms of the [EFTL](https://www.eclipse.org/legal/tck.php).
- [x] By checking this box I attest that all TCK requirements have been met, including any compatibility rules.

**Comment by jhanders34** [comment]
Ratifying CCR for Jakarta EE 11 Core Profile for use in the [Release Review issue for Core Profile](https://github.com/jakartaee/specifications/pull/782).

**Comment by brideck** [comment]
CCR has been updated with new Core Profile TCK SHA, as have the linked results pages.

**Comment by edburns** [comment]
Thanks, @brideck , this looks good to me. The TCK zip links are all good.

**Comment by ivargrimstad** [comment]
The ongoing mentor review of https://github.com/jakartaee/specifications/pull/782 has resulted in a new build of the TCK.

**Comment by starksm64** [comment]
Restaged, https://www.eclipse.org/downloads/download.php?file=/ee4j/jakartaee-tck/jakartaee11/staged/eftl/jakarta-core-profile-tck-11.0.0.zip

```
***********************************************************************************
***                        TCK bundle information                               ***
*** Name:       jakarta-core-profile-tck-11.0.0.zip                                     ***
*** Bundle Copied to URL:    http://download.eclipse.org/ee4j/jakartaee-tck/jakartaee11/staged/eftl//jakarta-core-profile-tck-11.0.0.zip ***
*** Date and size: date: 2024-11-14 18:20:10.444442915 +0000, size(b): 224670        ***
*** SHA256SUM: 0357bfab7025972edb2bf50277b6b4206b499a2961bc94e783f34782cc4a9bda ***
***                                                                             ***
***********************************************************************************
```

**Comment by Emily-Jiang** [comment]
@brideck please rerun the tests with the new zip and update the test records.

**Comment by brideck** [comment]
> @brideck please rerun the tests with the new zip and update the test records.

Already done. Just waiting on doc publication to update the CCR.

**Comment by brideck** [comment]
CCR has been updated with newest Core Profile TCK SHA, as have the linked results pages.
